modules/update-service-install-cli.adoc

```diff
@@ -0,0 +1,121 @@
+[id="update-service-install-cli_{context}"]
+= Installing the {product-title} Update Service by using the CLI
+
+You can use the {product-title} CLI to install the {product-title} Update Service Operator.
+
+.Procedure
+
+To install the {product-title} Update Service Operator by using the {product-title} CLI:
+
+. Create a Namespace for the {product-title} Update Service Operator:
+
+.. Create a Namespace object YAML file, for example, `update-service-namespace.yaml`, for the {product-title} Update Service Operator:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: FIXME-default-namespace
+  annotations:
+    openshift.io/node-selector: ""
+  labels:
+    openshift.io/cluster-monitoring: "true" <1>
+----
+<1> Set the `openshift.io/cluster-monitoring` label to enable operator recommended cluster monitoring on this namespace.
+
+.. Create the Namespace:
++
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
+----
++
+For example:
++
+[source,terminal]
+----
+$ oc create -f update-service-namespace.yaml
+----
+
+. Install the {product-title} Update Service Operator by creating the following objects:
```

Comment:
From my testing, if these are created separately you must create the OperatorGroup before the Subscription or the operator does not deploy.